changelog.md

```diff
@@ -1,3 +1,9 @@
+3.5.11.3 (unreleased)
+========
+- @ulidtko
+    - [#362](https://github.com/bitemyapp/esqueleto/pull/362)
+      - Add `updateReturning`, `deleteReturning` as Postgres extensions.
```

Comment:
This implementation adds a field to the `SideData` type, which unfortunately does make this a breaking change and would need to go out as `3.6.0.0`.

However, I've suggested an alternative implementation that should be strictly additive, allowing it to be `3.5.12.0`.